src/client/src/services/blotterService.js

```diff
@@ -9,12 +8,16 @@ var _log:logger.Logger = logger.create('BlotterService');
 
 export default class BlotterService extends ServiceBase {
 
-  constructor(serviceType:string, connection:Connection, schedulerService:SchedulerService, referenceDataService:ReferenceDataService) {
+  constructor(serviceType:string,
+              connection:Connection,
+              schedulerService:SchedulerService,
+              referenceDataService:ReferenceDataService,
+              openFin:OpenFin) {
     super(serviceType, connection, schedulerService);
     this._tradeMapper = new TradeMapper(referenceDataService);
   }
 
-  getTradesStream() : Rx.Observable<Array<Trade>> {
+  getTradesStream() : Rx.Observable<TradesUpdate> {
```

Comment:
you should import this type, if and when we turn on flow it'll expect it imported